rust-escrow/conditional-escrow/src/lib.rs

```diff
@@ -62,6 +62,15 @@ impl ConditionalEscrow {
         }
     }
 
+    pub fn proportion_deposit_of(&self, payee: &AccountId) -> Balance {
+        match self.deposits.get(payee) {
+            Some(deposit) => {
+                deposit * 1000 / self.funding_amount_limit
+            },
+            None => 0,
+        }
+    }
```

Comment:
If we are setting the `claim` method inside the `FT` contract, I think this method should also belong to the `FT` contract. 

The `FT` contract will query `get_deposits_of` each corresponding `ConditionalEscrow` to get the proportion.